www/include/Administration/brokerPerformance/brokerPerformance.ihtml

```diff
@@ -103,3 +109,9 @@
     }
 </script>
 {/literal}
+
+{literal}
+    <script>
+        formatDateMoment();
+    </script>
+{/literal}
```

Comment:
It is not possible to call the function directly after the bracket? This will avoid closing the "script" and "literal" tags to open them again afterwards.

Reply:
change done